src/variants/VariantsAdmin.ts

```diff
@@ -45,4 +45,10 @@ export class VariantsAdmin {
     }
     return (await api.post(`/applications/${appId}/${variant.type}`, variant)).data as Variant;
   }
+
+  async delete(api: AxiosInstance, appId: string, filter?: VariantFilter) {
+    const variants = await this.find(api, appId, filter);
+    const promises = variants.map(variant => api.delete(`/applications/${appId}/${variant.variantID!}`));
+    return Promise.all(promises);
```

Comment:
this can become a one liner:
```suggestion
    return Promise.all(
      (await this.find(api, appId, filter))
        .map(variant => api.delete(`/applications/${appId}/${variant.variantID!}`))
    );
```